File: CSqlManager/CSqlManager/Brand.cs
namespace CSqlManager;

public class Brand
{
    public string Code { get; set; }
    public string Name { get; set; }

    public Brand(string code, string name)
    {
        Code = code;
        Name = name;
    }

    public override string ToString()
    {
        return Name;
    }
}

File: front/front/src/app/model/brand.ts

export class Brand {
    _name: string;
    _code: string;

    constructor(data: any) {
        this._code = data.code;
        this._name = data.name;
    }
    
    public get code(): string {
        return this._code;
    }

    public set code(s: string) {
        this._code = s;
    }

    public get name(): string {
        return this._name;
    }

    public set name(name: string) {
        this._name = name;
    }
}